src/_layouts/default.html

```diff
@@ -1,12 +1,17 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    {% include meta.html %} {% include styles.html %} {%include analytics.html %}
+    {% include meta.html %} {% include styles.html %} {% include analytics.html %}
   </head>
 
   <body>
     {% include header.html %}
-    <main class="container">{{content}}</main>
+    <main class="container">{{ content }}</main>
     {% include footer.html %}
+
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-ENjdO4Dr2bkBIFxQpeoTz1HIcje39Wm4jDKdf19U8gI4ddQ3GYNS7NTKfAdVQSZe"

```

Comment:
Part of the above change to the version

```suggestion
      integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4"
```